test/integration/serverless/run.sh

```diff
@@ -200,6 +203,8 @@ for function_name in "${all_functions[@]}"; do
     done
     printf "\e[A\e[K" # erase previous log line
 
+    echo $raw_logs > $RAWLOGS_DIR/$function_name
```

Comment:
I find it helpful for debugging to have access to the raw logs.

Reply:
I agree, it's helpful having access to raw logs. Would a temp file be better here to ensure we're cleaning up our raw logs (or, rather, the OS cleans up raw log files for us)? Something like this should work: 

```
RAWLOGS_DIR=$(mktemp -d)
echo "Raw logs will be written to ${RAWLOGS_DIR}"
```